tests/10-verify.spec.js

```diff
@@ -349,11 +349,78 @@ describe('verify API (presentations)', () => {
   });
 });
 
+describe('tests for multiple credentials', async () => {
+
+  const credentialsCount = [5, 25, 50, 100];
+
+  for(const count of credentialsCount) {
+    it('cause error when credentials are tampered', async function() {
+      this.timeout(10000);
+      const challenge = uuid();
+      const {presentation, suite: vcSuite, documentLoader} =
+        await _generatePresentation({challenge, credentialsCount: count});
+
+      // tampering with the first two credentials id
+      presentation.verifiableCredential[0].id = 'some fake id';
+      presentation.verifiableCredential[1].id = 'some other fake id';
+
+      const result = await vc.verify({
+        documentLoader,
+        presentation,
+        suite: vcSuite,
+        unsignedPresentation: true
+      });
+
+      const credentialOne = result.credentialResults[0];
+      const credentialTwo = result.credentialResults[1];
+      const credentialThree = result.credentialResults[2];
+      const firstErrorMsg = result.credentialResults[0].error.errors[0].message;
+
+      result.verified.should.be.a('boolean');
+      result.verified.should.be.false;
+      credentialOne.verified.should.be.a('boolean');
+      credentialOne.verified.should.be.false;
+      credentialTwo.verified.should.be.a('boolean');
+      credentialTwo.verified.should.be.false;
+      credentialThree.verified.should.be.a('boolean');
+      credentialThree.verified.should.be.true;
+      firstErrorMsg.should.contain('Invalid signature.');
```

Comment:
It's good that you tested that the ones that were expected to fail have failed.  Also need to test that the remaining items were verified successully.  See my not below.